src/betterproto/plugin/models.py

```diff
@@ -653,8 +653,9 @@ def __post_init__(self) -> None:
             self.output_file.typing_imports.add("AsyncIterable")
             self.output_file.typing_imports.add("Iterable")
             self.output_file.typing_imports.add("Union")
-        if self.server_streaming:
-            self.output_file.typing_imports.add("AsyncIterator")
+
+        # Required by both client and server
+        self.output_file.typing_imports.add("AsyncIterator")
```

Comment:
This will import `AsyncIterator even when both `self.client_streaming` and `self.server_streaming` are both false (e.g. for normal requests of type `rpc X(A) returns B`.

This should really be:

```suggestion
        if self.client_streaming or self.server_streaming:
            self.output_file.typing_imports.add("AsyncIterator")
```